apisix/cli/etcd.lua

```diff
@@ -347,6 +347,11 @@ function _M.init(env, args)
                 util.die(errmsg)
             end
 
+            if res_put:find("CommonName of client sending a request against gateway", 1, true) then
+                errmsg = str_format("etcd client cert auth cannot be used with gRPC-gateway\n")
```

Comment:
Also tell users the solution, such as "Please configure the usernamd and password in configuration file".

Reply:
done